### ISSUE TITLE
Check failures with current RSQLite, again

### ISSUE THREAD

**Issue by krlmlr** [dbottomly/poplite#3]
Follow-up to #2.

I'm now getting for the CRAN version:

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘poplite-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: Database-class
> ### Title: Class '"Database"'
> ### Aliases: Database Database-class columns,Database-method dbFile
> ###   dbFile,Database-method populate populate,Database-method schema
> ###   schema,Database-method tables,Database-method isOpen,Database-method
... 26 lines ...
+  head(dbReadTable(examp.con, "team_franch"))
+  
+  dbDisconnect(examp.con)
+  
+ }
Loading required package: Lahman
Loading required package: RSQLite
Error in rsqlite_connection_valid(dbObj@ptr) : 
  external pointer is not valid
Calls: populate ... dbIsValid -> dbIsValid -> rsqlite_connection_valid -> .Call
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  4: .local(obj, ...)
  5: isOpen(obj)
  6: isOpen(obj)
  7: dbIsValid(con@connection)
  8: dbIsValid(con@connection)
  9: rsqlite_connection_valid(dbObj@ptr) at /home/muelleki/git/R/RSQLite/R/SQLiteConnection.R:64
  
  testthat results ================================================================
  OK: 115 SKIPPED: 0 FAILED: 3
  1. Error: Database population (@test-poplite.R#452) 
  2. Error: Querying with Database objects (@test-poplite.R#567) 
  3. Error: sample tracking example but with direct keys between dna and samples (@test-poplite.R#801) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

The following objects are masked from ‘package:base’:

    intersect, setdiff, setequal, union

Loading required package: DBI

... 8 lines ...

    filter

Error in makeSchemaFromData(dna, "dna") : 
  ERROR: The names of the supplied data.frame need to be modified for the database see correct.df.names

Error: processing vignette 'poplite.Rnw' failed with diagnostics:
 chunk 8 
Error in rsqlite_connection_valid(dbObj@ptr) : 
  external pointer is not valid
Execution halted
```

Can you replicate this? I'm having trouble running the tests with you current development version 2d2bfb40e. Thanks.

**Comment by dbottomly** [comment]
Ok, should be fixed in 06400b6, main issue with dbIsValid probably related to https://github.com/rstats-db/RSQLite/issues/217

**Comment by krlmlr** [comment]
Thanks for the nudge, will fix soon and submit an update to CRAN. Will open a new issue for any new failures.
